help/edit-or-delete-a-message.md

```diff
@@ -38,58 +56,81 @@ content.
 
 ## Delete a message
 
-Editing a message to delete its content will cause the message to be
-displayed as **(deleted)**.  The original sender and timestamp of the
-message will still be displayed, and the original content of the
-message is still accessible via Zulip's [edit
-history](/help/view-a-messages-edit-history) feature.  This can be the
-least confusing option for other users.
+### Delete message content
+
+Editing a message to delete its content will cause the message to be displayed
+as **(deleted)**.  The original sender and timestamp of the message will still
+be displayed, and the original content of the message is still accessible via
+Zulip's [edit history](/help/view-a-messages-edit-history) feature.  This can be
+the best option for avoiding confusion if other users have already responded to
+your message.
+
+{start_tabs}
+
+{tab|desktop-web}
+
+{!message-actions.md!}
+
+1. Click the **pencil** (<i class="fa fa-pencil"></i>) icon. If you do not see
+   the **pencil** (<i class="fa fa-pencil"></i>) icon, you do not have
+   permission to delete the content of this message.
+
+1. Delete the content of the message.
+
+1. Click **Save**.
+
+{tab|mobile}
+
+{!message-long-press-menu.md!}
+
+1. Tap **Delete message** to delete the content of the message.  If you do not
+   see the **Delete message** option, you do not have permission to delete the
+   content of this message.
+
+{end_tabs}
 
 ### Delete a message completely
```

Comment:
Similarly, maybe we can change this to a level 2 subheading and then either delete or update the next subheading accordingly.